src/components/Button/index.js

```diff
@@ -364,6 +364,9 @@ const propTypes = {
   download: PropTypes.string,
   extend: PropTypes.bool,
   href: PropTypes.string,
+  /**
+   * Name of the icon. All [icons](/?path=/docs/components-icon) are supported.
+   */
   icon: PropTypes.oneOfType([PropTypes.string, PropTypes.node, PropTypes.func]),
   iconPosition: PropTypes.oneOf(['left', 'right']),
   innerRef: PropTypes.func,
```

Comment:
What is this used for ?

Reply:
Same for : ` displayProgressOnlyl` When must say that user need to have `progress` prop too too make it works

Reply:
`innerRef` seems to be a ref you could associate with button child but as we pass props into it we can directly do `ref={aRef}`